src/tests/ftest/util/server_utils.py

```diff
@@ -216,6 +324,236 @@ def create_yaml(self, filename):
             raise ServerFailed(
                 "Error writing daos_server command yaml file {}: {}".format(
                     filename, error))
+        return filename
+
+
+class ServerManager(ExecutableCommand):
+    """Defines object to manage server functions and launch server command."""
+    # pylint: disable=pylint-no-self-use
```

Comment:
Is this still needed?  Unless I missed it, I didn't see a class method that doesn't use `self`